packages/components/docs/localDev.md

```diff
@@ -181,24 +181,31 @@ update the `package.json` file and commit that change.  Then you can do a `git p
 Note that for some of our premium npm packages, we are using `npm` to do the build/publish. In those cases, you'll need
 to update the package.json file version number manually before running `npm publish`.
 
-## Merging changes into develop
+## Merging labkey-ui-components changes into develop
 
-1. Message the Frontend dev room chat about starting the pull request merge. This is to make sure two people aren't
-merging at the same time which might result in conflicting package version numbers.
 1. Do one final merge of the `develop` branch into your feature branch for `labkey-ui-components`.
-1. Run one final lint of your changes, `yarn run lint-branch-fix`, and review the changes applied.
-1. Update the `releaseNotes/labkey/components.md` file with what will be your release version number and release date.
-1. Run the commands to build, test, and publish: `yarn build`, `yarn test`, `yarn publish`.
-1. Push your final set of commits from `labkey-ui-components` to github so that TeamCity can do a final run of the jest tests.
+2. Run one final lint of your changes, `yarn run lint-branch-fix`, and review the changes applied.
+3. Update the `releaseNotes/labkey/components.md` file with what will be your release version number and release date.
+4. Run the commands to build and test: `yarn build`, `yarn test`.
+5. Push your final set of commits from `labkey-ui-components` to GitHub so that TeamCity can do a final run of the jest tests.
+6. Message the Frontend dev room chat about starting the pull request merge. This is to make sure two people aren't
+   merging at the same time which might result in conflicting package version numbers.
+7. Check on the [TeamCity](https://teamcity.labkey.org) build status and jest test status (also shown in the PR).
+8. Merge the pull requests for `labkey-ui-components`.
+9. Run the command to publish: `yarn publish`.
+10. Message the Frontend dev room chat that the merge is complete.
+
+## Reference the new labkey-ui-components version in LabKey modules
```

Comment:
This feels a little odd to me to be separated out as a separate heading. The text in the steps are written as if it was an extension of the previous steps / process. Having it as a separate heading seems like it is a process that could be done independently, but really it fits in with the publish process for a components package merge (in my opinion).